lib/commander/index.js

```diff
@@ -0,0 +1,1231 @@
+/**
```

Comment:
This file was copied verbatim from https://github.com/tj/commander.js/blob/v2.15.0/index.js.